patches/server/0953-fixed-npe.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: NonSwag <mrminecraft00@gmail.com>
+Date: Tue, 6 Dec 2022 23:04:21 +0100
+Subject: [PATCH] fixed npe
```

Comment:
This belongs in a separate PR. You seemed to have closed your other PRs for this?